ArcadiaParties-SPA/src/app/user/user-info/user-info.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex-container">
+  <mat-card>
+    <mat-card-title style="text-align:center">Your information</mat-card-title>
```

Comment:
It is preferable to not have mixed styles: inline and in css files.

Reply:
I've moved all styles into css file.